### ISSUE TITLE
Accessibility - Is the theme too Dark? Should we consider having a dark mode option

### ISSUE THREAD

**Issue by JamesRexMiller4** [JamesRexMiller4/refactortractor#18]
I was thinking that it might be cool to have a Dark Mode option like you are starting to see for Slack, web browsers, Twitter. I don't think it would be too hard. We may need to have a separate Sass file that has the White Theme. 

Definitely an extension but I think this would be a cool exercise to try out. @RayRedGoose thoughts?

**Comment by JamesRexMiller4** [comment]
I did this changing three different css properties in the dev tools. <5 min
![image](https://user-images.githubusercontent.com/27719824/67064095-6d851400-f126-11e9-9569-43a3fecbf1dc.png)
